src/cli.rs

```diff
@@ -2,15 +2,23 @@ use std::io;
 
 use tokio::signal::ctrl_c;
 
-use crate::{cliapp::make_app, scheduler::run_scheduler};
+use crate::{cliapp::make_app, logging, scheduler::run_scheduler};
 
-pub async fn execute() -> io::Result<()> {
```

Comment:
Per our documentation

> In a multithreaded application, spawned threads should inherit the Hub from the main thread. In order for that to happen, the Sentry client must be initialized before starting an async runtime or spawning threads. This means you'll have to avoid using macros such as #[tokio::main] or #[actix_web::main], because they start the runtime first.